roles/cockpit-ui/tasks/main.yml

```diff
@@ -45,16 +23,15 @@
   oc_route:
     name: docker-registry
     namespace: default
-    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
+    kubeconfig: "{{ openshift_master_config_dir }}/admin.kubeconfig }}"
     state: list
   register: docker_registry_route
-  when: check_docker_registry_exists.rc == 0
 
 - name: Retrieve cockpit kube
   oc_route:
     name: registry-console
     namespace: default
-    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
+    kubeconfig: "{{ openshift_master_config_dir }}/admin.kubeconfig }}"
     state: list
   register: registry_console_cockpit_kube
   when: check_docker_registry_exists.rc == 0
```

Comment:
No need for the `when: check_docker_registry_exists.rc == 0` on the Deploy registry-console task below.